packages/renderer/src/lib/table/Table.svelte

```diff
@@ -143,9 +143,9 @@ function setGridColumns() {
       class="grid grid-table gap-x-0.5 mx-5 h-12 bg-charcoal-800 hover:bg-zinc-700 rounded-lg mb-2"
       animate:flip="{{ duration: 300 }}"
       role="row">
-      <div class="whitespace-nowrap justify-self-start"></div>
+      <div class="whitespace-nowrap justify-self-start" role="cell"></div>
       {#if row.info.selectable}
-        <div class="whitespace-nowrap place-self-center">
+        <div class="whitespace-nowrap place-self-center" role="cell">
```

Comment:
It would be really helpful, if every cell element, would have associated `aria-label` that would help to identify what kind of information/object does this particular cell contains.

In this case: `role="cell" aria-label="Toggle image"`
Although this behaviour would be more visible in other cells.